### PR TITLE
Rename the default branch to main

### PR DESCRIPTION
**Pull request by ysksuzuki**
Signed-off-by: Yusuke Suzuki <yusuke-suzuki@cybozu.co.jp>

### CODE REVIEW

File: v1/Makefile [hunk 1]
```diff
@@ -11,7 +11,7 @@ DOCDIR := $(WORKDIR)/usr/share/doc/placemat
 EXAMPLEDIR := $(WORKDIR)/usr/share/doc/placemat/examples
 BASH_COMPLETION_DIR := $(WORKDIR)/etc/bash_completion.d
 BINDIR := $(WORKDIR)/usr/bin
-VERSION = 1.1.0-master
```
Comment (ymmt2005) [inline v1/Makefile:14]: I guess this is the version name for debian packages, not a git tag.

**Review by ymmt2005 — CHANGES_REQUESTED** [review]
Please fix the commit according to my comments.

File: v1/Makefile (hunk 1 above)
Comment (ysksuzuki) [reply v1/Makefile:14]: Placemat V1 seems to have used a git tag for VERSION, so I used the same settings as here.
https://github.com/cybozu-go/placemat/blob/v1.5.3/.circleci/config.yml#L61

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Pull request merged**